src/prng/mod.rs

```diff
@@ -98,12 +118,13 @@
 //! CSPRNGs are a little different in that they typically generate a block of
 //! output in a cache, and pull outputs from the cache. This allows them to have
 //! good amortised performance, and reduces or completely removes the influence
-//! of surrounding code on the CSPRNG performance. It does however cause *poor
-//! worst case performance*.
+//! of surrounding code on the CSPRNG performance.
 //!
-//! ## State size
+//! ### Worst-case performance
+//! Because CSPRNGs usually produce a block of values into a cache, they have
+//! poor worst case performance (in contrast to regular PRNGs).
 //!
-//! Normal PRNGs often use very little memory, commonly only a few words, where
+//! Simple PRNGs often use very little memory, commonly only a few words, where
 //! a *word* is usually either `u32` or `u64`. This is not universal however,
```

Comment:
Okay — since this starts about "simple PRNGs", change "This is not universal" to "This is not true for all non cryptographic PRNGs"

Or don't use *Simple*. I don't know, but doesn't seem like quite a lot of non-crypto PRNGs are *simple*.